### DISCUSSION TITLE
Select shadow not working

### DISCUSSION THREAD

**Discussion by erwinstone** [mantinedev/mantine#3722] [Q&A] [answered]
https://stackblitz.com/edit/react-ts-8pqgsq?file=App.tsx

https://user-images.githubusercontent.com/96282452/223899029-97fb55fb-4cde-4987-bd0f-46879eef44c6.mp4

**Comment by rtivital** [comment, marked as answer]
Select component does not support shadow prop. You can customize dropdown styles with Styles API.

**Reply by erwinstone** [reply]
If so, then it's best to remove the shadow property from the documentation.
https://mantine.dev/core/select/?t=props

**Reply by mattaningram** [reply]
Couldn't it be made to support shadow prop by changing the hard coded value here? https://github.com/mantinedev/mantine/blob/5c1d77209f92fb14377d75c7cdc8d36f84705b63/src/mantine-core/src/Select/Select.tsx#L546
